src/mail-app/mail/view/MailView.ts

```diff
@@ -1023,6 +1023,7 @@ export class MailView extends BaseTopLevelView implements TopLevelView<MailViewA
 	private async trashSelectedMails() {
 		const actionableMails = await this.mailViewModel.getResolvedActionableMails()
 		trashMails(mailLocator.mailModel, actionableMails)
+		this.mailViewModel.removeStickyMail(actionableMails)
```

Comment:
is this the only place? should move also trigger it?

Reply:
I think this is already handled by the "Behavior after moving mail" setting, so the mail will be no longer displayed.

Reply:
Actually, I thought about it more and you are right